lib/Service/RecordingService.php

```diff
@@ -156,17 +167,23 @@ public function getContentFromFileArray(array $file, Room $room, Participant $pa
 			throw new InvalidArgumentException('invalid_file');
 		}
 
-		$content = file_get_contents($file['tmp_name']);
-		unlink($file['tmp_name']);
+		$resource = fopen($file['tmp_name'], 'r');
+		$resourceStat = fstat($resource);
```

Comment:
`fopen` and `fstat` can return `false` in case of failure. Although it should not happen I think it would be good to check if they failed and throw an exception in that case.

Reply:
I'm not that familiar with the totality of the spreed/Talk source code. Is there a recommended exception that should be used in those cases, or the built-in ones should suffice (UnexpectedValueException)?

Reply:
The rest of the method just uses generic `InvalidArgumentException` with a descriptive parameter, so that could be (ab)used again in this case :-) (even if it does not exactly match [the official description of the exception](https://www.php.net/manual/en/class.invalidargumentexception.php))

Reply:
Got it. Thanks.